linux_os/guide/system/permissions/restrictions/sysctl_kernel_perf_event_paranoid/rule.yml

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disallow kernel profiling by unprivileged users'
+
+description: '{{{ describe_sysctl_option_value(sysctl="kernel.perf_event_paranoid", value="2") }}}'
+
+rationale: |-
+    Kernel profiling can reveal sensitive information about kernel behaviour.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 81054-9
+
+references:
+    anssi: NT28(R25)
```

Comment:
This rule actually fulfills `NT28(R23)`.
And since this reference is here, please enable it for RHEL7, including the CSV file, and select it in `rhel7/profiles/anssi_nt28_intermediary.profile`

Reply:
`NT28(R23)` instead of `NT28(R25)`, or `NT28(R23)` in addition to `NT28(R25)`?

Reply:
`NT28(R23)` instead of `NT28(R25)`.
Sorry I wasn't clear.